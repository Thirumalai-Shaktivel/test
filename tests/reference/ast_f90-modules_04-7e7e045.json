{
    "basename": "ast_f90-modules_04-7e7e045",
    "cmd": "lfortran --show-ast-f90 --no-color {infile}",
    "infile": "tests/../integration_tests/modules_04.f90",
    "infile_hash": "c2a3f2488f0cbfc5efc150ea277129289bfb30dc2a42d33d4861357b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast_f90-modules_04-7e7e045.stdout",
    "stdout_hash": "8778744f554b3710b17f2323a9a026e79c3a32e3452f53c14604be35",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}