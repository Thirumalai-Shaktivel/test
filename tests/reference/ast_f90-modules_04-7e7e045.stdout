module a_04
implicit none

contains

subroutine b()
print *, "b()"
end subroutine b

end module a_04

program modules_04
use iso_fortran_env
implicit none
call f()

contains

subroutine f()
use a_04, only: b
call b()
end subroutine f

integer function g()
use a_04, only: b
call b()
g = 5
end function g

end program modules_04
